test/fixtures/config/globalSearch.json

```diff
@@ -11,7 +11,8 @@
         "backgroundColor": "",
         "enableContinueWatching": true,
         "enableHeader": true,
-        "enableGlobalSearch": true
+        "enableGlobalSearch": true,
+        "enableInVideoSearch": true
```

Comment:
Global search and in video search should have their own config fixture. These are two separate features.